package.json

```diff
@@ -31,7 +31,7 @@
     "firebase-tools": "^3.0.8",
     "google-closure-compiler": "^20160713.2.0",
     "google-closure-library": "^20160713.0.0",
-    "google-closure-templates": "^20150411.0.0",
+    "google-closure-templates": "^20150410.0.0",
```

Comment:
A bug in Catapult caused this to incorrectly get changed to `1.0.0` in https://github.com/firebase/firebaseui-web/commit/acadb105e56376ec6947f24b2d63cc1651d1f7c4. Thankfully, it doesn't affect the `$ npm install firebaseui` call so the deploy is okay. It did however cause Travis to fail forthe previous commit. I'll make sure to fix the bug in Catapult for the next release.